client/components/navigation/Navbar.tsx

```diff
@@ -46,69 +53,172 @@ function Navbar() {
+            h="100%"
+            w="100%"
+            display={{ sm: 'flex', md: 'none' }}
+            bg="white"
+            spacing={10}
+            fontSize="3xl"
+            pt={20}
+          >
+            <Link
+              color="black"
+              p="0.45rem"
+              m="0 0.25rem"
+              cursor="pointer"
+              textDecoration="none"
+              borderRadius="18px"
+              href="/gigs"
+            >
+              Gigs
+            </Link>
+            <Link
+              color="black"
+              p="0.45rem"
+              m="0 0.25rem"
+              cursor="pointer"
+              textDecoration="none"
+              borderRadius="18px"
+              href="/jobs"
+            >
+              Jobs
+            </Link>
 
-        <Link
-          color="black"
-          p="0.45rem"
-          m="0 0.25rem"
-          cursor="pointer"
-          textDecoration="none"
-          _hover={{textDecoration: "none", bgColor:"#e2e8f0"}}
-          _focus={{textDecoration: "none", border:"none"}}
-          borderRadius="18px" href="/developers">Devs</Link>
-        <Link
-          color="black"
-          p="0.45rem"
-          m="0 0.25rem"
-          cursor="pointer"
-          textDecoration="none"
-          _hover={{textDecoration: "none", bgColor:"#e2e8f0"}}
-          _focus={{textDecoration: "none", border:"none"}}
-          borderRadius="18px"
-          href="/companies">Companies</Link>
-        <Link
-          color="black"
-          p="0.45rem"
-          m="0 0.25rem"
-          cursor="pointer"
-          textDecoration="none"
-          _hover={{textDecoration: "none", bgColor:"#e2e8f0"}}
-          _focus={{textDecoration: "none", border:"none"}}
-          borderRadius="18px"
-          href="/about">About</Link>
-        </HStack>
+            <Link
+              color="black"
+              p="0.45rem"
+              m="0 0.25rem"
+              cursor="pointer"
+              textDecoration="none"
+              borderRadius="18px"
+              href="/developers"
+            >
+              Devs
+            </Link>
+            <Link
+              color="black"
+              p="0.45rem"
+              m="0 0.25rem"
+              cursor="pointer"
+              textDecoration="none"
+              borderRadius="18px"
+              href="/companies"
+            >
+              Companies
+            </Link>
+            <Link
+              color="black"
+              p="0.45rem"
+              m="0 0.25rem"
+              cursor="pointer"
+              textDecoration="none"
+              borderRadius="18px"
+              href="/about"
+            >
+              About
+            </Link>
+            <Link textDecoration="none" href="/auth">
+              <ButtonBlack>Sign Up</ButtonBlack>
+            </Link>
+          </VStack>
+        )}
 
+        <HStack
+          spacing={{ base: 2, sm: 7 }}
+          display={{ sm: 'none', md: 'block' }}
```

Comment:
Since `base` isn't defined, very small screens will still render this component.

```suggestion
          display={{ base: 'none', md: 'block' }}
```

What this means is that `display="none"` until we reach the `md` breakpoint at which point it becomes `display="block"`.

Reply:
More info on responsive styles in Chakra: https://chakra-ui.com/docs/features/responsive-styles